release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -242,6 +242,182 @@ To support this feature, OLM now enforces CRD upgrades are safer by ensuring
 existing CRD storage versions are not missing in the upgraded CRD, avoiding
 potential data loss.
 
+[id="ocp-4-5-operator-api"]
+==== Read-only Operator API (Technology Preview)
+
+The new Operator API is now available as a Technology Preview feature in
+read-only mode. Previously, installing Operators using Operator Lifecycle
+Manager (OLM) required cluster administrators to be aware of multiple APIs,
+including CatalogSources, Subscriptions, ClusterServiceVersions, and
+InstallPlans. This single Operator API resource is a first step towards a more
+simplified experience discovering and managing the lifecycle of Operators in a
+{product-title} cluster.
+
+Currently only available using the CLI and requiring a few manual steps to
+enable, this feature previews interacting with Operators as a first-class API
+object. Cluster administrators can discover previously installed Operators using
+this API in read-only mode.
+
+In a future release of {product-title}, cluster administrators will be able to
+use the Operator API to install an Operator bundle in a cluster without being
+part of a catalog, shortening the route for developers to test their packaging.
+
+To enable this Technology Preview feature:
+
+.Procedure
+
+. Disable xref:../architecture/architecture-installation.adoc#unmanaged-operators_architecture-installation[Cluster Version Operator (CVO) management] of the OLM:
++
+----
+$ oc patch clusterversion version \
```

Comment:
when I test it, I take method in https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusterversion.md
```
If there are currently no other overrides configured:

$ cat <<EOF >version-patch-first-override.yaml
- op: add
  path: /spec/overrides
  value:
  - kind: DaemonSet
    group: apps/v1
    name: cluster-network-operator
    namespace: openshift-cluster-network-operator
    unmanaged: true
EOF

To add to list of already existing overrides:

$ cat <<EOF >version-patch-add-override.yaml
- op: add
  path: /spec/overrides/-
  value:
    kind: DaemonSet
    group: apps/v1
    name: cluster-network-operator
    namespace: openshift-cluster-network-operator
    unmanaged: true
EOF
$ oc patch clusterversion version --type json -p "$(cat version-patch.yaml)"
```